src/Symfony/Bridge/Doctrine/Form/DoctrineOrmTypeGuesser.php

```diff
@@ -53,6 +53,8 @@ public function guessType($class, $property)
 
         switch ($metadata->getTypeOfField($property)) {
             case Type::TARRAY:
+            case Type::SIMPLE_ARRAY:
+            case Type::JSON_ARRAY:
```

Comment:
Can you remove `JSON_ARRAY` so that we can merge this PR?

Reply:
Done.